src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php

```diff
@@ -76,12 +76,63 @@ public function printOpenTag(Formatter $formatter, FeatureNode $feature, Scenari
         /** @var JUnitOutputPrinter $outputPrinter */
         $outputPrinter = $formatter->getOutputPrinter();
 
-        $outputPrinter->addTestcase(array(
-            'name' => $name,
+        $outputPrinter->addTestcase($this->addCircleCiAttributes(array(
+            'name' => $this->addLineNumberToScenarioName($name, $scenario->getLine()),
+            'file' => $this->addLineNumberToFeatureFile($this->convertToRelativePath($feature->getFile()), $scenario->getLine()),
             'classname' => $feature->getTitle(),
             'status' => $this->resultConverter->convertResultToString($result),
             'time' => $this->durationListener ? $this->durationListener->getDuration($scenario) : ''
-        ));
+        )));
+    }
+
+    /**
+     * Append scenario line number in the feature file to a scenario name
+     *
+     * @param string $name
+     * @param integer $lineNumber
+     * @return string
+     */
+    private function addLineNumberToScenarioName(string $name, int $lineNumber) {
+        return $name . " Line #" . $lineNumber;
+    }
+
+    /**
+     * Append scenario line number in the feature file to the feature file
+     *
+     * This can be used to reference the scenario when running tests in the command line
+     *
+     * @param string $path
```

Comment:
Let's either add a description to this or tweak the method description to be more obvious:

```suggestion
     * @param string $path the path to the feature file
```

```
     * Append the line number of the scenario within the feature file to the feature file's path
```

Reply:
I renamed this method and updated the comments to make it clearer what it is doing.